### PR TITLE
Dom manipulation

### PR DESCRIPTION
**Pull request by J-C-S-V**
Here is what we did on this project:

- Create elements using DOM manipulations
-  Add eventListener to the hamburguer menu
- Change style using JavaScript

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
+const hamburguerMenuIcon = document.querySelector('.three-bar-menu');
+// const crossIcon = document.querySelector('.crossIcon');
```
Comment (rloterh) [inline index.js]: - [x] The mobile menu (X) button, does not disappear when clicked. I can see that the DOM selector of this button is commented out. Please ensure to improve upon this aspect

File: index.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  const crossIcon = document.createElement('a');
+  // const bottomLine = document.createElement('span');
+
+  const headerBar = document.querySelector('.header-bar');
+  const mainSection = document.querySelector('.main-section');
+  // let mainContainerProjects = document.querySelector(
+  //   '.main-container-projects'
+  // );
+
+  // add a class to the anchor tag
```
Comment (rloterh) [inline index.js]: - [x] [OPTIONAL]
There are commented code blocks found in your code on the repository. Kindly endeavor to remove **all** commented code from your code before pushing it to the repository so as to keep your code well organized.

File: index.html [hunk 3]
```diff
@@ -5,6 +5,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="index.css" />
+    <script src="index.js" defer></script>
     <title>Juank's portifolio</title>
   </head>
   <body>
```
Comment (rloterh) [inline index.html:11]: - [x] [OPTIONAL]
 To enhance the UX, kindly ensure that the _hamburger_ and the _close_ button are in the same position when toggled.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Juank, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦  🏃 

## Highlights 👍  
- Menu items jump to the corresponding part of the page when clicked!
- Mobile menu icon is hidden on the desktop screen! 
- Loom video on project features, FAQs, and objectives is awesome! 👌 

## Required Changes ♻️
_Kindly check the inline comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

_Kindly check the inline comments marked **[OPTIONAL]** under the review._

**You can also consider:**
- Kindly include appropriate instructions in your ReadMe on how to _setup_ and _run_ your app, under the _Getting Started_ section.
Implementing the above change(s) will make your ReadMe/ repository stand out even better! 😉

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (J-C-S-V) [reply index.html:11]: Hello @rloterh ,

Thank you so much for your feedback. Could you explain in more detail when you say "kindly ensure that the hamburger and the close button are in the same position when toggled"?.

Thanks in advance!

**Review by tugsem — CHANGES_REQUESTED** [review]
## STATUS: REQUIRED CHANGES ♻️
Hello @J-C-S-V and @dhy-612  👋 , 

You've done well implementing some of the requested changes, but there are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text

 ## Highlights 🚀
✔️Good job on making the requested changes

 ## Required Changes ♻️

- [x] Please check this video presentation and fix the close button when a link is clicked and make sure it's disappeared. Also when it's appeared, it must cover the page and not allow horizontal scrolling.
https://app.flonnect.com/view/video/saranlitugsem/Flonnect_2023-03-07_b4a86d36-19b3-4b9e-864f-de0dd1c78d05
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (rloterh) [reply index.html:11]: @J-C-S-V 

Let's say, when you click the _hamburger menu_ and it disappears, the **close buttons** should be within that same spot where the mouse pointer clicked the **hamburger menu**. In other words, the _hamburger menu_ and the _close button_ should be on the same _x_ and _y_ coordinates.

Cheers! 😃

**Comment by J-C-S-V** [comment]
Hello @tugsem,

Could you paste the resource where it says that requirement is mandatory? I think that's optional since It's not described in the milestone requirements:

https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md

Thanks in advance.

**Review by nicupop729 — COMMENTED** [review]
Hello @J-C-S-V  👋,

_This is an additional review as a result of the second opinion request._

Kudos for the good effort put into place for this milestone.

As we found in the project requirements, the mobile menu should disappear when the user clicks on one of the options.
image

In order to reach the next milestone, please make sure to implement this important feature. 

Keep rocking 🚀!

**Comment by J-C-S-V** [comment]
Thank you so much @nicupop729 !!!!

That explanation was much clearer 🙌

I'm going to work on that then.

**Review by brainconnect93 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @J-C-S-V, 👋 👋  

Your project is complete! Great job for following the project requirement🥇and implementing the changes requested by the previous reviewers. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 




## Good Points 👍

- Has a proper Pull Request. ✔️
- No Linter errors ✔️
- Good use of Github flow ✔️

## Optional changes

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**